info/v1/container.go

```diff
@@ -520,6 +520,33 @@ type FsStats struct {
 	WeightedIoTime uint64 `json:"weighted_io_time"`
 }
 
+type AcceleratorStats struct {
+	// Make of the accelerator (nvidia, amd, google etc.)
+	Make string `json:"make"`
+
+	// Model of the accelerator (tesla-p100, tesla-k80 etc.)
+	Model string `json:"model"`
+
+	// ID of the accelerator.
+	ID string `json:"id"`
+
+	// Total accelerator memory.
+	// unit: bytes
+	MemoryTotal uint64 `json:"memory_total"`
+
+	// Total accelerator memory allocated.
+	// unit: bytes
+	MemoryUsed uint64 `json:"memory_used"`
+
+	// Percent of time over the past sample period during which
+	// the accelerator was actively processing.
+	DutyCycle uint64 `json:"duty_cycle"`
```

Comment:
Is this common jargon for GPU utilization?  It doesn't seem intuitive to me, but I also am also not very familiar with GPUs...

Reply:
To clarify: Is "DutyCycle" something most people will know is the utilization of their GPU?

Reply:
Yeah, duty cycle is commonly used to refer percent of time in which a system was active.